storage/rocksdb/rdb_datadic.h

```diff
@@ -326,6 +328,12 @@ class Rdb_key_def {
     PER_PARTITION_CF_FLAG = 4,
   };
 
+  // bit flags which denote myrocks specific fields stored in the record
+  // currently only used for TTL.
+  enum INDEX_FLAG {
+    TTL_FLAG = 1,
```

Comment:
Spoken earlier, probably need to add a static array for the sizes of each index flag field. Also change this from "TTL_FLAG = 1" to "TTL_FLAG = 1 << 0" so that it's explicit these should be bitfields.